src/vlindex/verilog_indexer.cpp

```diff
@@ -0,0 +1,1457 @@
+/*******************************************************************\
+
+Module: Verilog Indexer
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+#include "verilog_indexer.h"
+
+#include <util/cmdline.h>
+#include <util/cout_message.h>
+#include <util/suffix.h>
+#include <util/unicode.h>
+
+#include <verilog/verilog_parser.h>
+#include <verilog/verilog_preprocessor.h>
+#include <verilog/verilog_y.tab.h>
+
+#include <algorithm>
+#include <fstream>
+#include <iosfwd>
+#include <iostream>
```

Comment:
Likely not both of these, and preferably none of them?

Reply:
Removed iosfwd.